lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode/Battery.pm

```diff
@@ -86,4 +93,65 @@ sub _parseDate {
     return;
 }
 
+sub _mergeBatteries {
```

Comment:
This can be done like that but sometimes equality test is not enough. When there is only one battery, if upower returns first one battery and then also dmidecode returns one battery, i have considered that it is the same battery. Because we need to have the number of all batteries returned by dmidecode, we cannot merge the elements one by one.